hw/dv/dpi/usbdpi/usbdpi.c

```diff
@@ -195,13 +195,14 @@ void usbdpi_device_to_host(void *ctx_void, const svBitVecVal *usb_d2p) {
 
       case ST_IDLE:
         // Nothing to do
-        ctx->state = ST_GET;
         break;
 
       default:
         assert(!"Invalid/unknown state");
         break;
     }
+
+    ctx->state = ST_GET;
```

Comment:
Just checking, but did the change from [83597373cc5db85bc3c527b95b24ecde3a1015ab](https://github.com/lowRISC/opentitan/commit/83597373cc5db85bc3c527b95b24ecde3a1015ab#diff-953b4db380c5ee57046af37ad5d330efb1d3998b8a56d069a3538c756977a49bR198) not work out?

Reply:
I'm afraid it's more a case of too many branches, PRs and tests and confusion with a merge. I shall change it to what we've been using with the ASIC t-l smoke test and generated byte enables and check that the Verilator t-l sims are unaffected still; changing state to ST_GET when it was not ST_IDLE would only assist in trying to stumble on after an error that should not have occurred, but I should perhaps have left myself a comment to remind myself of that!

Reply:
Restored the previous change and left myself/posterity a comment for next time!